### ISSUE TITLE
`/api/views/search/messages` does not obey `time_zone` argument, always returns UTC

### ISSUE THREAD

**Issue by drewmiranda-gl** [Graylog2/graylog2-server#15030]
`/api/views/search/messages` does not obey `time_zone` argument, always returns UTC

This via the api-browser, this endpoint shows that it supports a `time_zone` parameter. However, this parameter appears to have no effect on the data it returns. Timestamps are always in UTC.

![image](https://user-images.githubusercontent.com/107503402/227220241-f76a6a56-0b00-456f-8bdd-4a52cd4d80d6.png)

![image](https://user-images.githubusercontent.com/107503402/227220406-73d9a107-dd32-4e0a-9762-524d10fb7843.png)

## Expected Behavior

If a timezone argument/parameter is listed as supported, the output should honor it and returned data should be in that timezone.

## Current Behavior

Timestamp is always returned as UTC regardless of what `time_zone` is set to.

## Steps to Reproduce (for bugs)

Use the API browser to send a req, such as:

`/api/views/search/messages`

```json
{
    "timerange": {
        "type": "absolute",
        "from": "2023-03-23T13:23:27.286Z",
        "to": "2023-03-23T13:28:27.287Z"
    },
    "limit": 10,
    "time_zone": "America/Chicago"
}
```

## Context

This was posed as a question via https://community.graylog.org/t/the-timestamp-returned-by-the-graylog-api-is-abnormal/28178 . I did some testing and validation and confirmed that either the openapi info is wrong and this endpoint doesn't support timezone, or it is not applying the timezone parameter.

Let me know if there are any questions, if you need any additional information, or need me to test/validate anything further. Happy to help! 🚀 

## Your Environment

* Graylog Version: 5.0.5
* Java Version: Bundled
* Elasticsearch Version: OpenSearch 2.4.1
* MongoDB Version: 5.0.14
* Operating System: Ubuntu Server 20.04 LTS
* Browser version: Chrome 111.0.5563.110

**Comment by Joel-Duffield-Graylog** [comment]
It seems like this issue from the community could be related as well. Agregations in the search screen are ignoring timezone. https://community.graylog.org/t/aggregation-timestamp-reverts-to-utc-although-timestamp-field-is-utc-1/28237

**Comment by drewmiranda-gl** [comment]
> It seems like this issue from the community could be related as well. Agregations in the search screen are ignoring timezone. https://community.graylog.org/t/aggregation-timestamp-reverts-to-utc-although-timestamp-field-is-utc-1/28237

Not 100% sure this is the same issue or not (so possibly we may need a separate issue? Will have to defer to dev/eng. on this one). For clarity though:

# What

When using aggregations (e.g. metrics via Graylog) on the timestamp field, the timezone offset of the user is ignored and timestamps are only displayed in their raw original UTC format.

![image](https://user-images.githubusercontent.com/107503402/227966252-ef2a516c-0a3c-48b9-b0c8-08aeec34a901.png)

# Expected behavior
Anywhere a timestamp is, its display using the user's declared timezone offset preference. For this specific example, the aggregated timestamps have the user's offset applied.

# Current behavior
Timestamp is displayed as UTC

# Your Environment
* Graylog Version: 5.0.5
* Java Version: Bundled
* ~~Elasticsearch~~ OpenSearch Version: OpenSearch 2.6
* MongoDB Version: 6.0.4
* Operating System: Ubuntu Server 22.04 LTS
* Browser version: Chrome 111.0.5563.110

**Comment by janheise** [comment]
ok, the first one should be fixed with #15296, but the second one is unrelated
